woocommerce/woocommerce
# Remove the navigation nudge note

### All Submissions:






### Changes proposed in this Pull Request:

Removes the navigation nudge note as it's no longer needed.

Closes #33129 .

### How to test the changes in this Pull Request:

1. Spin up a new site with this branch of WC
2. Do not enable the new navigation
3. Visit the homescreen
4. Make sure the navigation note ("You now have access to the WooCommerce navigation") is not added

### Other information:




### FOR PR REVIEWER ONLY:
